src/main/java/com/youtube/tracker/feign/YoutubeTrackerFeign.java

```diff
@@ -10,5 +10,5 @@
 public interface YoutubeTrackerFeign {
 
     @GetMapping("/v3/search")
-    ResponseEntity<YoutubeSearchResponse> search(@RequestParam String part, @RequestParam String channelId, @RequestParam String key, @RequestParam("q") String keyword);
+    ResponseEntity<YoutubeSearchResponse> search(@RequestParam String part, @RequestParam String channelId, @RequestParam String key, @RequestParam("q") String keyword, String publishedAfter);
```

Comment:
RequestParam missing.
Also break this into two lines

Reply:
oops sorry